### ISSUE TITLE
v38 planning

### ISSUE THREAD

**Issue by viceice** [renovatebot/renovate#27992]
### Describe the proposed change(s).

- [x] ~~#18811~~
- [x] #27535
  - [x] #28193
- [x] #27985
- [x] #28308
- [x] Require Node v20 #30291
- [ ] Bump base image #30373
  - [x] https://github.com/renovatebot/base-image/pull/164
  - [x] https://github.com/renovatebot/base-image/pull/208
  - [x] https://github.com/renovatebot/base-image/pull/277
  - [x] https://github.com/renovatebot/base-image/pull/281
  - [x] https://github.com/renovatebot/base-image/pull/358
  - [x] https://github.com/renovatebot/base-image/pull/393
  - [x] https://github.com/renovatebot/base-image/pull/424
- [x] https://github.com/renovatebot/renovate/pull/28588
- [x] #29447
- [x] #29525

### Deferred to future major

- [ ] #6842
- [ ] #16359 - blocked due to json5 migration
- [ ] #21442 - blocked by renovateCompatibility (#19980)
- [ ] https://github.com/containerbase/base/pull/1991
- [ ] #21446
- [ ] #23910

### Related issues
- #25608
- #23911
- #21134

**Comment by rarkins** [comment]
@viceice any major updates to embedded containerbase tools?

**Comment by HonkingGoose** [comment]
I copy/pasted the items from the _Deferred to future major_ list into the v39 planning issue:

- #25608
